src/Tribe/Field.php

```diff
@@ -481,7 +481,7 @@ public function textarea() {
 			$field .= $this->do_field_name();
 			$field .= $this->do_field_attributes();
 			$field .= '>';
-			$field .= esc_html( stripslashes( $this->value ) );
+			$field .= esc_textarea( html_entity_decode( stripslashes( $this->value ) ) );
```

Comment:
I think we need more comments explaining this and justification for it with an example.